### ISSUE TITLE
Fail on gene list upload

### ISSUE THREAD

**Issue by vasiliosz** [robinandeer/puzzle#93]
Returns server error on gene list upload. The file is uploaded however, and the list is there when you go back and reload the page. 

```
[2016-02-04 09:05:03,668] ERROR in app: Exception on /genelists [POST]
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1882, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1537, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1441, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1535, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1521, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/vaszac/proj/puzzle/puzzle/blueprints/public/views.py", line 107, in gene_list
    case_ids=case_ids)
UnboundLocalError: local variable 'case_ids' referenced before assignment
[2016-02-04 09:05:03,668] puzzle.factory            ERROR    Exception on /genelists [POST]
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1882, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1537, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1441, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1535, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1521, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/vaszac/proj/puzzle/puzzle/blueprints/public/views.py", line 107, in gene_list
    case_ids=case_ids)
UnboundLocalError: local variable 'case_ids' referenced before assignment
[2016-02-04 09:05:03,669] werkzeug                  INFO     127.0.0.1 - - [04/Feb/2016 09:05:03] "POST /genelists HTTP/1.1" 500 -
```

**Comment by moonso** [comment]
We fixed this yesterday, not sure if @robinandeer pushed it?

**Comment by vasiliosz** [comment]
I'm on master @ b9bd5473df12b31483d148ae2ae4f8c7a400be62

**Comment by moonso** [comment]
Ok let's just wait for @robinandeer patch

**Comment by moonso** [comment]
Fixed in #102
